pkgs/development/tools/documentation/doxygen/default.nix

```diff
@@ -19,16 +30,24 @@ stdenv.mkDerivation rec {
   ];
 
   buildInputs = [ libiconv ]
+    ++ lib.optionals withSqlite [ sqlite.dev ]
```

Comment:
```suggestion
    ++ lib.optionals withSqlite [ sqlite ]
```
this should work, too

Reply:
Adopted change locally; will be fixed next time I force-push.